sdk/src/wallet/account/operations/transaction/high_level/create_account.rs

```diff
@@ -10,20 +10,20 @@ use crate::{
         output::{
             feature::MetadataFeature,
             unlock_condition::{GovernorAddressUnlockCondition, StateControllerAddressUnlockCondition},
-            AliasId, AliasOutputBuilder, Output,
+            AccountId, AccountOutputBuilder, Output,
         },
     },
     wallet::account::{types::Transaction, Account, OutputData, TransactionOptions},
 };
 
-/// Params `create_alias_output()`
+/// Params `create_account_output()`
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct CreateAliasParams {
-    /// Bech32 encoded address which will control the alias. Default will use the first
+pub struct CreateAccountParams {
+    /// Bech32 encoded address which will control the account. Default will use the first
     /// address of the account
```

Comment:
Also not really great, but maybe
```suggestion
    /// Bech32 encoded address which will control the account. Default will use the first
    /// ed25519 address of the wallet account
```